hat/assets/js/apps/Iaso/utils/usersUtils.ts

```diff
@@ -62,7 +63,7 @@ export const getDisplayName = (
     // Some endpoint have user_name and some username (without the _, fun)
     const userName = user.user_name ?? user?.username;
     if (!user.first_name && !user.last_name) {
-        return userName;
+        return userName || '';
     }
     return `${userName} (${user.first_name ? `${user.first_name}` : ''}${
```

Comment:
if `userName === ''`,  this will return `(firstName, lastName)`. We should probably add a check to remove the partheses when `!userName`

Reply:
`userName ` cannot be ''